### PR TITLE
Add result, start, output pageobjects

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

File: integrations/standalone/tests/integration/parts/start.ts [hunk 1]
```diff
@@ -1,54 +1,72 @@
...
-    await page.getByLabel('Signature').clear();
-    await CollapseUtil.open(page, 'Input parameters');
-    await TableUtil.removeRow(page, 0);
-    await TableUtil.fillExpression(page, 0, '');
-    await CodeEditorUtil.focus(page);
-    await CodeEditorUtil.clear(page);
+
+  async clear() {
+    await this.signature.clear();
+    await this.paramSection.toggle();
```
Comment (weissreto) [inline integrations/standalone/tests/integration/parts/start.ts:56]: Do you need this line?

File: integrations/standalone/tests/integration/parts/start.ts [hunk 2]
```diff
@@ -1,54 +1,72 @@
...
+    await this.params.clear();
+    await this.mapping.row(1).column(2).edit('');
```
Comment (weissreto) [inline integrations/standalone/tests/integration/parts/start.ts:58]: fill or clear

**Review by weissreto — APPROVED** [review]
Some comments. Otherwise good

File: integrations/standalone/tests/integration/parts/start.ts (hunk 1 above)
Comment (ivy-lli) [reply integrations/standalone/tests/integration/parts/start.ts:56]: the param section / collapsable is per default closed. Was a wish by bb...

File: integrations/standalone/tests/integration/parts/start.ts [hunk 3]
```diff
@@ -1,54 +1,72 @@
...
+    await this.mapping.row(1).column(2).fill('"bla"');
+    await this.code.fill('code');
   }
-  async assertFill({ page }: Part) {
-    await expect(page.getByLabel('Signature')).toHaveValue('myStart');
-    await CollapseUtil.open(page, 'Input parameters');
+
+  async assertFill() {
+    await this.signature.expectValue('myStart');
+    await this.paramSection.toggle();
```
[the remark this answers is not in the document]
Comment (ivy-lli) [reply integrations/standalone/tests/integration/parts/start.ts:43]: the param section / collapsable is per default closed. Was a wish by bb...

**Pull request merged**